archunit-junit/junit5/engine/src/test/java/com/tngtech/archunit/junit/ArchUnitTestEngineTest.java

```diff
@@ -463,20 +478,80 @@ void tags_of_test_classes() {
             );
         }
 
+        @Test
+        void meta_tag_of_test_classes() {
+            EngineDiscoveryTestRequest discoveryRequest = new EngineDiscoveryTestRequest().withClass(TestClassWithMetaTag.class);
+
+            TestDescriptor descriptor = testEngine.discover(discoveryRequest, engineId);
+
+            TestDescriptor testClass = getOnlyElement(descriptor.getChildren());
+            assertThat(testClass.getTags()).containsOnly(TestTag.create("tag-one"), TestTag.create("tag-two"));
```

Comment:
I'm not sure if its a good idea to unify them, as the only reason they are the same is because the `TestClassWithMetaTag`/`TestClassWithMetaTags` test classes that I created have the same `@ArchTag`s as the existing `TestClassWithTags` test class. They are not dependent on each other, and can be modified separately without affecting each other whatsoever. Maybe I should make the test classes more distinct so that they don't appear the same?

Reply:
I have renamed the `@ArchTag`s in the new test classes to distinguish them from the existing test classes, so that the assertions no longer appear the same